client/storage-monitor/src/lib.rs

```diff
@@ -42,22 +45,22 @@ pub struct StorageMonitorParams {
 	/// Required available space on database storage. If available space for DB storage drops below
 	/// the given threshold, node will be gracefully terminated. If `0` is given monitoring will be
 	/// disabled.
-	#[arg(long = "db-storage-threshold", value_name = "MB", default_value_t = 1000)]
+	#[arg(long = "db-storage-threshold", value_name = "MiB", default_value_t = 1000)]
```

Comment:
When you start using base 2, may also use `1024` here so the default is one `GiB` and not some odd mixture.